### ISSUE TITLE
Fix unhelpful error for request /full/full/0/default.jpg with v3.0

### ISSUE THREAD

**Issue by zimeon** [zimeon/iiif#43]
```
> ./iiif_testserver.py 
root: Installing pil IIIFManipulator at /1.0_pil/ v1.0 none
root: Installing pil IIIFManipulator at /1.1_pil/ v1.1 none
root: Installing pil IIIFManipulator at /2.0_pil/ v2.0 none
root: Installing pil IIIFManipulator at /2.1_pil/ v2.1 none
root: Installing pil IIIFManipulator at /3.0_pil/ v3.0 none
root: Setup server on http://localhost:8000/ ...
 * Serving Flask app "iiif_testserver" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
werkzeug:  * Running on http://localhost:8000/ (Press CTRL+C to quit)
```

the access <http://localhost:8000/3.0_pil/starfish/full/full/0/default> and the error is unhelpfully:

```
IIIF Image Server Error

Illegal size value (not enough values to unpack (expected 2, got 1)).

parameter=size
code=400
```

which is likely because it looks for a comma if not `max`. Probably should know about `full` and point out the specific issue to be helpful to any developer/tester.

**Comment by zimeon** [comment]
The HTML index page for 3.0 had `full` changed to `max` in 1e60949b3169afbe48c50c1f4dd314788ca36f4c

**Comment by zimeon** [comment]
Closed by #47
